From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Sat, 1 Nov 2025 13:08:38 +0100
Subject: [PATCH] HACK: fix building with gcc 15

Without this, the check will incorrectly fail, because the arguments
don't match.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 acinclude.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/acinclude.m4 b/acinclude.m4
index 9cf9483e5592..89835d7f053c 100644
--- a/acinclude.m4
+++ b/acinclude.m4
@@ -609,7 +609,7 @@ GMP_PROG_CC_WORKS_PART([$1], [long long reliability test 1],
 
 #if defined (__GNUC__) && ! defined (__cplusplus)
 typedef unsigned long long t1;typedef t1*t2;
-void g(){}
+void g(int, const t1*, t1, t1*, const t1*, int){}
 void h(){}
 static __inline__ t1 e(t2 rp,t2 up,int n,t1 v0)
 {t1 c,x,r;int i;if(v0){c=1;for(i=1;i<n;i++){x=up[i];r=x+1;rp[i]=r;}}return c;}
